app/Todo.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Todo extends Model
+{
+    protected $fillable = [
+        'title', 'description', 'priority', 'flag'
+    ];
+
+    public function user() 
+    {
+        return $this->hasOne('App\User', 'user_id');
```

Comment:
ne treba ti drugi parametar, jer ce laravel sam skontati da treba da veze za `user_id`, da ti se recimo model drugacije zvao, tipa Member , a treba da vezes za user_id onda bi morala staviti, a kada se isto zovu onda laravel to sam handle-a